### PR TITLE
Fixed the English translations, added some Dutch translations (WIP)

### PR DESCRIPTION
**Pull request by dizaztor**
Fixed the English translations, added some Dutch translations (WIP).

I will continue translating the English strings to Dutch tomorrow, and see if I have made any grammar errors :).

But for the English part, I fixed some translations.

### CODE REVIEW

**Comment by dizaztor** [comment]
I will proofread them one more time (either today or tomorrow), and see if I'm missing anything, just to be sure.

**Comment by obinslab** [comment]
Thanks for your greate work, especial for the improvement of en.yml. Meanwhile, I am a bit confused about `ing`.

**Comment by dizaztor** [comment]
For an example:
> Write data to application processor...

Here you're not saying what's happening, this looks more like a command that the user has to do, but:

> Writing data to application processor...

Implies that the program is busy writing the data to the processor.

**Comment by obinslab** [comment]
For your example, I agree. 

Others, for example:
> common.operate.failed: Operate Failed
> common.operate.success: Operate Success
> common.add.success: Add Success
> common.add.failed: Add Failed
> common.delete.success: Delete Success
> common.delete.failed: Delete Failed
> common.save.success: Save Success

I'd like to translate these to
> common.operate.failed: Operated Failed
> common.operate.success: Operated Success
> common.add.success: Added Success
> common.add.failed: Added Failed
> common.delete.success: Deleted Success
> common.delete.failed: Deleted Failed
> common.save.success: Saved Success

**Comment by dizaztor** [comment]
Hmm, if you say so. But I could tell you that they don't sound right. For an example:

> Added Failed

How is it "added" if it has failed? If you choose "adding", then you could say, for an example, that the program tried _adding_, but it failed, so "_Adding $X Failed_", which is just "_Adding Failed_"

But if you still want them this way, then I don't mind changing the translations. :)

**Comment by obinslab** [comment]
Hi Sir
Can you illustrate your question with some pictures?I don't quite understand what you mean because of the language.



Best Regards

Email: service@obins.net
Twitter: https://twitter.com/obinslab
 
From: Mario
Date: 2019-05-03 23:21
To: obinslab/obinslab-starter-translations
CC: obinslab; Comment
Subject: Re: [obinslab/obinslab-starter-translations] Fixed the English translations, added some Dutch translations (WIP) (#13)
For an example:
Write data to application processor...
Here you're not saying what's happening, this looks more like a command that the user has to do, but:
Writing data to application processor...
Implies that the program is busy writing the data to the processor.

**Comment by obinslab** [comment]
Today, I found an awesome website. 
> An Android developer spent two years collecting more than 15 million app translations and making a search site, hoping to help independent developers complete at lower cost and higher quality internationalization work.

Search `add successfully` have 216 results.
Search `added successfully` have 1459 results.
Search `adding successfully` have 27 results.

> https://i18ns.com/index.html?q=add%20successfully
> https://i18ns.com/index.html?q=added%20successfully
> https://i18ns.com/index.html?q=adding%20successfully

So I think, the best choice should be `Added successfully`

**Comment by dizaztor** [comment]
Mine was not `adding successfully`, but `adding success`. If you're going with the `ing` path, then this should be indeed changed to `adding successful`. 

`added successfully` does the job as well. Your choice. Which one should I change it to?

**Comment by obinslab** [comment]
I did some research again on https://i18ns.com

App `com.tencent.wework` from Tencent used translations like this:
> Added successfully
> Adding failed

I think it's perfect. Thank you very much for your patience.

**Comment by dizaztor** [comment]
Yeah, that's what I had in mind as well. I'm on it, will push in a few minutes!

**Comment by dizaztor** [comment]
Let me know if you still want to improve some other stuff!

**Comment by obinslab** [comment]
Just one problem:

`# NOT COMPLETE YET` in `nl.yml`.

If you have finished, please remove it.

Thanks!

**Comment by obinslab** [comment]
One more thing, `common.copy_text` should keep as `Copy`.
This string will show like this:
copy-text

File: locales/nl.yml [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+kbd.firmware.upgrade.status.success: Firmware upgraden gelukt.
+
+# Main Window, LEDs
+kbd.led: Light
+kbd.led.effect.group: Light Group
+kbd.led.effect.group.reset: Reset
+kbd.led.effect.group.delete.confirm_content: Weet je zeker dat je deze lichteffectgroep wilt verwijderen?
+kbd.led.effect.list: Lichtlijst
+kbd.led.effect.list.limit.annepro1_three: Je kan alleen maar 3 lichteffecten per 1 groep hebben 
+kbd.led.effect.list.limit.annepro2_sixteen: Je kan alleen maar 6 lichteffecten per 1 groep hebben
```
Comment (obinslab) [inline locales/nl.yml]: Here should be `kbd.led.effect.list.limit.annepro2_sixteen: Je kan alleen maar 16 lichteffecten per 1 groep hebben`

Comment (dizaztor) [reply locales/nl.yml]: Ah, right, didn't notice. Thanks, I will fix it.

**Comment by obinslab** [comment]
We had a activity on Twitter: https://twitter.com/obinslab/status/1119572238854803457
We want to send you a ANNE PRO 2 as a gift.
Please contact us by email, our email is:service@obins.net

Best regards!

**Comment by obinslab** [comment]
We had a activity on Twitter: https://twitter.com/obinslab/status/1119572238854803457
We want to send you a ANNE PRO 2 as a gift.
Please contact us by email, our email is:service@obins.net

Best regards!

**Comment by dizaztor** [comment]
@obinslab Is this better now? Am I still missing something?

Also, I've sent you guys an email (dizaztor@outlook.com). Thanks a lot! :)

**Pull request merged**